cl/search/management/commands/import_harvard_pdfs.py

```diff
@@ -51,6 +51,13 @@ def add_arguments(self, parser):
             help="Directory for reading crosswalk files",
             required=True,
         )
+        parser.add_argument(
+            "--job",
+            type=str,
+            choices=["import_pdf"],
+            default="import_pdf",
+            help="",
+        )
```

Comment:
I see other things running through the crosswalk file to do work (#4614). I’ve no sound opinion for what the “correct” approach is. `jobs` here is driven by my interest in not copy/pasting (and then owning) the moderate complexity surrounding crosswalk processing. @jtmst, I’m happy to take any direction that you can give.